variables.tf

```diff
@@ -53,6 +53,8 @@ variable "buckets" {
 variable "cloudtrail" {
   type = object({
     enabled                       = optional(bool, false)
+    enable_cloudwatch_logs        = optional(bool, false)
+    cloud_watch_logs_group_name   = optional(string, "aws-cloudtrail-logs")
```

Comment:
can we create cloudwatch log group based on cloudtrail name instead of passing it from outside as variable?
something like this 
```
"${variable.cloudtrail.name}-cloudtrail-logs"